suite/tests/CMakeLists.txt

```diff
@@ -3972,10 +3972,11 @@ if (UNIX)
   endif ()
   # i#784: test app behavior on alarm
   tobuild(linux.alarm linux/alarm.c)
-  # XXX i#2043: enable for A64 once append_fcache_enter_prologue() is finished.
-  if (NOT APPLE AND NOT ANDROID AND NOT AARCH64) # Test uses Linux-specific timer code.
-    tobuild(linux.signal_race linux/signal_race.c)
-    target_link_libraries(linux.signal_race rt)
+  if (NOT APPLE AND NOT ANDROID) # Test uses Linux-specific timer code.
+    if (NOT AARCH64) # TODO i#2043: Enable this for AArch64.
```

Comment:
There seems to be some other issue with linux.signal_race app. It continues to hang when run under DR, even with this PR.

Reply:
Based on the comments on both the test sources, `signal_racesys` is the one that "adjusts (the timer syscall) to arrive immediately **before** a system call" whereas, `signal_race` "adjusts it to arrive immediately **after** the return from the system call"

`linux.signal_racesys` seemed the relevant one to reproducing the issue in #4271.

Reply:
But the comment says #2043: so that is no longer the right issue?

Does the hang go away with -unsafe_build_ldstex?

Reply:
> But the comment says #2043: so that is no longer the right issue?

Probably not. I'll change it to #1569.

> Does the hang go away with -unsafe_build_ldstex?

It continues to hang.